### PR TITLE
docs: FeedView templates

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Updating Documentation 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md). (for bug fixes / features) 
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by nickrandolph** [comment]
@ajpinedam is this still being worked on? Can we finish this off in preparation for next release

**Comment by agneszitte** [comment]
> @ajpinedam is this still being worked on? Can we finish this off in preparation for next release

@ajpinedam can we have a little update for this PR please

File: doc/Overview/Mvux/FeedView.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ The `Data` property provides access to the value reported by the last message re
 </Page>
 ```
 
-In the `Text` property binding `Data.Name` of the above example, `Data` is a property of the `FeedViewState` accessible in the template, which gets the most recent `Contact` from the server, and then we're binding to that `Contact`'s `Name` property.
+In the `Text` property binding `Data.` `Name` of the above example `Data` is a property of the `FeedViewState` accessible in the template, which gets the most recent `Contact` from the server, and then we're binding to that `Contact`'s `Name` property.
```
Comment (nickrandolph) [inline doc/Overview/Mvux/FeedView.md:85]: Why add space here?

**Comment by nickrandolph** [comment]
@agneszitte @ajpinedam it would be good to get this PR finished in time for the 5.2 docs - lots of small tidy up changes

File: doc/Overview/Mvux/FeedView.md (hunk 1 above)
Comment (ajpinedam) [reply doc/Overview/Mvux/FeedView.md]: This was a mistake. Fixed.

**Comment by ajpinedam** [comment]
> @agneszitte @ajpinedam it would be good to get this PR finished in time for the 5.2 docs - lots of small tidy up changes

Update and ready to be reviewed!

**Review by dr1rrb — APPROVED** [review]

**Pull request merged**